src/reconcile/rec_write.c

```diff
@@ -613,8 +613,8 @@ __rec_init(WT_SESSION_IMPL *session, WT_REF *ref, uint32_t flags, WT_SALVAGE_COO
      * Update the page state to indicate that all currently installed updates will be included in
      * this reconciliation if it would mark the page clean.
      *
-     * Add a write barrier to make it more likely that a thread adding an update will see this state
-     * change.
+     * Add a release barrier to make it more likely that a thread adding an update will see this
+     * state change.
```

Comment:
I see a full barrier after. It says increase the chance to see this change, which is not a valid use case for barriers.

Reply:
b06cc6cec

Reply:
Why we still say ensuring visibility?

Reply:
Where specifically?